devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json

```diff
@@ -156,6 +156,8 @@
         "gradle-plugin-version": "${quarkus-gradle-plugin.version}",
         "supported-maven-versions": "${supported-maven-versions}",
         "proposed-maven-version": "${proposed-maven-version}",
+        "supported-java-versions": "${supported-java-versions}",
+        "proposed-java-version": "${proposed-java-version}",
```

Comment:
@ia3andy how does this look to you?

Reply:
@gastaldi @aloubyansky 👆

Reply:
A Version Range as explained in https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html